BudgetSquirrel.Business/BudgetPlanning/Budget.cs

```diff
@@ -108,5 +149,10 @@ public void AddToFund(decimal amount)
         {
             this.FundBalance += amount;
         }
+
+        public bool IsOwnedBy(User user)

```

Comment:
Just a thought, we could in the future have some sort of joint or shared accounts. Where an account could be used by two people